lib/wallet/rpc.js

```diff
@@ -1877,31 +1877,44 @@ class RPC extends RPCBase {
   }
 
   async sendOpen(args, help) {
-    if (help || args.length < 1 || args.length > 2)
-      throw new RPCError(errs.MISC_ERROR, 'sendopen "name" ( force )');
+    if (help || args.length < 1 || args.length > 3)
+      throw new RPCError(errs.MISC_ERROR, 'sendopen "name" ( force ) ( account )');
```

Comment:
Putting the args in this order means `force` is required if you also want to pass `account`. Frankly I have no idea what `force` is doing: https://github.com/handshake-org/hsd/issues/93 but you may want to swap those

Reply:
You always add new arguments to the end, and you can totally call a function without defining intermediate arguments

Reply:
If you run this from CLI:
`hsw-rpc sendopen <name> <account>`
... won't it interpret the `<account>` argument as `<force>`?  I encountered similar issue with Braydon's wallet rpc updates. If you don't identify args specifically with `--account=`, etc, the CLI will interpret the args in order from left to right

Reply:
I guess we could make it a named param, but I was just thinking you could just pass `false` as the second arg

Reply:
Frankly, based on somewhat shallow review, I think `force` is actually unused, not sure what it was ever for, or if it's still supposed to even be there...?

Reply:
Idk what the intention is/was. Should probably ask jj